### PR TITLE
#80 - add codeblocks

### PR DESCRIPTION
**Pull request by JakubKermes**
Added often used functions into ready to use code blocks.  Still open for extension

### CODE REVIEW

File: src/Features/Traits/CodeBlocs.php [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+declare(strict_types=1);
+
+namespace Blumilk\BLT\Features\Traits;
+
+use Behat\Gherkin\Node\TableNode;
+use Blumilk\BLT\Features\Traits\Optional\SpatiePermission;
+use Symfony\Component\HttpFoundation\Response;
+
+trait CodeBlocs
```
Comment (EwelinaSkrzypacz) [inline src/Features/Traits/CodeBlocs.php:11]: Shouldn't it call "CodeBlocks"?

**Review by EwelinaSkrzypacz — APPROVED** [review]

**Review by krzysztofrewak — COMMENTED** [review]
Maybe `Common` would be better name?

**Comment by EwelinaSkrzypacz** [comment]
@JakubKermes will you change this name?

**Comment by JakubKermes** [comment]
rename to just `Common` or `CommonBlocks/CommonCode`?

**Comment by krzysztofrewak** [comment]
I don't see any advantage to name it "block".

**Review by EwelinaSkrzypacz — APPROVED** [review]

**Review by krzysztofrewak — APPROVED** [review]

**Pull request merged**